main/integration-tests/support/setup.js

```diff
@@ -80,16 +77,23 @@ class Setup {
     return session;
   }
 
-  async getDefaultIndexId() {
+  // For future enhancement, we can capture this in a different file similar to how we did the getGenerators() and getServices().
+  async getDefaultResources() {
     const adminSession = await this.defaultAdminSession();
-    const defaultProject = await adminSession.resources.projects.project(this.gen.defaultProjectId()).get();
-    return defaultProject.indexId;
-  }
+    const project = await adminSession.resources.projects.project(this.settings.get('projectId')).get();
 
-  async getDefaultAwsAccountId() {
-    const adminSession = await this.defaultAdminSession();
-    const defaultIndex = await adminSession.resources.indexes.index(this.defaultIndexId).get();
-    return defaultIndex.awsAccountId;
+    const indexId = project.indexId;
+    const index = await adminSession.resources.indexes.index(indexId).get();
+
+    const awsAccountId = index.awsAccountId;
+    const awsAccount = await adminSession.resources.awsAccounts.awsAccount(awsAccountId).get();
+
+    const stepTemplate = await adminSession.resources.stepTemplates
+      .versions('st-obtain-write-lock')
+      .version(1)
+      .get();
```

Comment:
I am wondering if we need this extra call.  None of the step template tests need any other property of the step template object, just the id.

Reply:
Just to keep it consistent like other returned resources.